views/mattermost/server_fact.view.lkml

```diff
@@ -82,100 +82,114 @@ view: server_fact {
                       ;;
   }
 
-  dimension: retention_28day_flag {
-    label: " 28-Day Retention"
-    group_label: " Telemetry Flags"
-    description: "Boolean indicating the instance was retained after 28 days since their first active date. This metric is a flag indicating users performed events between hour 672 and 696 from the instance's first active timestamp."
-    type: yesno
-    sql: COALESCE(${TABLE}.retention_28day_flag, false) ;;
+  dimension: retention_0day_flag {
+    label: "0-Day Retention"
+    group_label: "Telemetry Flags"
+    description: "Boolean indicating the instance was retained between (instance) creation time and 1 day since their first active date."
```

Comment:
Is this essentially saying that if there are ANY real time telemetry events that fire after server creation time (but before 1 day) then this is true?

Reply:
we are excluding the ones generated by the instance such as `category = performance` as of now. There was another category of events that were triggered by the instance, we need to update those as well, I will need your help review these events.

Reply:
last time I checked, there was one other one that gets triggered after 14 days of inactivity, called `inactive_server_email_sent`, a notification which goes via email, when clicked brings them back to the server. This was mostly seen in Cloud.

Reply:
I have the link for the events that get triggered right after an instance is created, send me a message on MM.